src/libraries/restore/runtime/runtime.depproj

```diff
@@ -73,6 +70,20 @@
     <Exec Command="chmod +x $(TestHostRootPath)%(DotnetExe.Filename)%(DotnetExe.Extension)" Condition="'$(TargetOS)' != 'Windows_NT' and '$(OS)' != 'Windows_NT'"/>
   </Target>
 
+  <Target Name="SetupRuntimePackNative"
```

Comment:
Given this target is specific to `Mono` should we include `Mono` in the target name?

Reply:
there are going to be some differences with desktop mono, so I'd like to address them in a separate PR.

Reply:
Do you have an issue tracking this work we can reference here?

Reply:
https://github.com/dotnet/runtime/issues/34203